src/common/const.ts

```diff
@@ -154,6 +155,7 @@ export const FIXED_DEVICE_CLASS_ICONS = {
   pressure: mdiGauge,
   reactive_power: mdiFlash,
   signal_strength: mdiWifi,
+  sound_pressure: mdiVolumeHigh,
```

Comment:
Another possibility is "mdi:ear-hearing"

Reply:
Yep, better! I like this one! 🙂

Reply:
done